services/booking-api/lambdas/getHistoricalAttendees.js

```diff
@@ -0,0 +1,39 @@
+import to from 'await-to-js';
+
+import * as response from '../../../libs/response';
+
+import booking from '../helpers/booking';
+
+export async function main(event) {
+  console.log('EVENT: ', JSON.stringify(event, undefined));
```

Comment:
You could use the logger helper function from libs?

Reply:
Will remove the console.log and create a subtask where we add logging of events in all booking-api lambdas in the same way.